modules/client/src/tokenVoting/internal/graphql-queries/proposal.ts

```diff
@@ -0,0 +1,80 @@
+import { gql } from "graphql-request";
+
+export const QueryTokenVotingProposal = gql`
+query tokenVotingProposal($proposalId: ID!) {
```

Comment:
The query from the file before starts uppercase, any reason to change the pattern here?